### PR TITLE
boost v1.67.0

### PR DESCRIPTION
**Pull request by tadeu**
Based on PR #52

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -7,11 +7,6 @@ os: osx
 osx_image: xcode6.4
 
 env:
-  matrix:
-    
-    - CONDA_PY=27
-    - CONDA_PY=35
-    - CONDA_PY=36
```
Comment (jakirkham) [inline .travis.yml]: Hmm...do we know why these disappeared?

File: appveyor.yml [hunk 2]
```diff
@@ -17,22 +17,6 @@ environment:
...
-      CONDA_PY: 35
-      CONDA_INSTALL_LOCN: C:\\Miniconda35-x64
-
-    - TARGET_ARCH: x86
-      CONDA_PY: 36
-      CONDA_INSTALL_LOCN: C:\\Miniconda36
-
-    - TARGET_ARCH: x64
-      CONDA_PY: 36
-      CONDA_INSTALL_LOCN: C:\\Miniconda36-x64
```
Comment (jakirkham) [inline appveyor.yml]: Same story here. :/

**Comment by jakirkham** [comment]
@conda-forge-admin, please rerender.

**Comment by jakirkham** [comment]
Thanks for picking this up, @tadeu.

Something is off with the re-rendering. So asked the bot for a new one. Let's see how that goes.

File: .travis.yml (hunk 1 above)
Comment (tadeu) [reply .travis.yml]: Good catch, I didn't check these other changes, I've just continued from where the bot PR stopped.

Comment (jakirkham) [reply .travis.yml]: Yeah, have seen these issues in the past with re-rendering. Though that is not a common occurrence with the re-rendering bot or manual re-renderings any more. Not sure why the autoupdate bot is running into this issue, but have seen it in a couple of cases. Normally our bot or manually re-rendering has fixed it afterwards. Could be anything from a connectivity issue or suppressed error.

Letting you know for awareness, @CJ-Wright. Probably not worth worrying about to much, but if there is an issue to track this that would be good for now. If you have any ideas, they would of course be welcome, but just being aware is probably good for now given cb3 and cs3 are rolling out too.

Comment (isuruf) [reply .travis.yml]: it was because `boost-cpp` PR was not merged at the time.

**Comment by jakirkham** [comment]
Seems the re-rendering bot fixed it. So no need to worry about it. 😌

File: .travis.yml (hunk 1 above)
Comment (jakirkham) [reply .travis.yml]: Good point. Thanks for mentioning that.

**Comment by tadeu** [comment]
Is there something else needed for the merge?

**Comment by jakirkham** [comment]
Thanks @tadeu. :)

**Pull request merged**